### ISSUE TITLE
Mac client trying to mount network drive and crashing

### ISSUE THREAD

**Issue by tizzo** [keybase/client#12263]
When I setup keybase on my new Mac I had a network drive mounted (a freenas smb share) and while it worked as long as the network drive was connected keybase now won't launch (or launches and freezes with a white box sans any interface elements) and repeatedly shows an error connecting to the network drive if I am whereI couldn't access the NAS.

<img width="423" alt="screen shot 2018-06-07 at 4 40 15 pm" src="https://user-images.githubusercontent.com/245728/41124984-8d9dfbcc-6a71-11e8-96ea-423f497dd69d.png">

**Comment by tizzo** [comment]
Bump bump.  This is driving me crazy. I've uninstalled and reinstalled keybase but to no avail - if I can't connect to the NAS keybase is non-functional (which is a drag as my team is using it for some of our internal communications).

**Comment by maxtaco** [comment]
@strib, how do you disable Fuse via config?

As a separate issue, it seems like there's a UI regression and there's no way to disable Fuse via UI, but that likely won't help here.

**Comment by strib** [comment]
`keybase uninstall -c redirector,fuse` plus restarting Keybase will disable the mount (or the Files tab in the Settings tab -- I suspect the regression is only in the admin builds)

But that's not really what the user is asking.  I believe there's some other problem that's been reported before where the osx mounter has to loop through the Finder favorites for some reason, and if one of them isn't responsive, it hangs.  I don't know anything about it though.

**Comment by maxtaco** [comment]
@strib you're right, it's still there, i didn't know it had moved.

**Comment by tizzo** [comment]
@strib @maxtaco sorry to poke this old issue but any chance y'all could provide some feedback on why keybase would be trying to remount a network share totally unrelated to keybase? That seems like a very strange and concerning behavior.

**Comment by strib** [comment]
Please see my comment above.  I don't think it's trying to mount it per se, it's just looping through the favorites list and macOS is trying to mount it for some reason during that loop.  Keybase isn't trying to look at the mount itself, just the favorites list.  But again I don't know anything about that code personally, sorry.

**Comment by tizzo** [comment]
@strib thank you so much for the snappy response. I misunderstood your above comment on first read.

Thanks again and thanks for the great product.  My team uses it all the time.

**Comment by tizzo** [comment]
@strib I spoke too soon - I also removed all favorites and cleared all history from my connect to server window. Nothing is listed there.  I tried running `keybase uninstall -c redirector,fuse`, tried uninstalling and reinstalling keybase, tried deleting application support - still can't launch keybase on my mac if I'm not on the LAN with this FreeNAS server - really annoying!

**Comment by tizzo** [comment]
@strib @maxtaco  any chance you lovely people have any other suggestions for a work around here?

**Comment by fluxens** [comment]
duplicate of #6359 ?

**Comment by tizzo** [comment]
@fluxens yeah, I think this is a dupe. It's also still plaguing me. Would love to see a fix as I can't find anyway to suppress this behavior despite all attempts to remove this drive from startup items and favorites.
